fabrictestbed/cli/cli.py

```diff
@@ -48,6 +49,22 @@ def __get_slice_manager(*, oc_host: str = None, cm_host: str = None, project_id:
     return SliceManager(oc_host=oc_host, cm_host=cm_host, project_id=project_id, scope=scope,
                         token_location=token_location)
 
+def __unpack(data) -> dict:
+    """
+    Unpacks embedded json turned from an objects to_dict() method. 
+    @param object_dict Dictionary returned from the Object.to_dict() method
```

Comment:
I am having trouble understanding this docstring. Perhaps it could be simplified so that simpletons can comprehend it? :-)

And that `param` is named `data` though?